packages/react/src/components/DataTable/TableToolbar.js

```diff
@@ -5,32 +5,45 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import cx from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { settings } from 'carbon-components';
 import { AriaLabelPropType } from '../../prop-types/AriaPropTypes';
 
 const { prefix } = settings;
 
-const TableToolbar = ({ children, ...rest }) => (
-  <section {...rest} className={`${prefix}--table-toolbar`}>
-    {children}
-  </section>
-);
+const TableToolbar = ({ children, size, ...rest }) => {
+  const className = cx({
+    [`${prefix}--table-toolbar`]: true,
+    [`${prefix}--table-toolbar--small`]: size === 'compact' || size === 'short',
+  });
+  return (
+    <section {...rest} className={className}>
+      {children}
+    </section>
+  );
+};
 
 TableToolbar.propTypes = {
   /**
    * Pass in the children that will be rendered inside the TableToolbar
    */
   children: PropTypes.node,
 
+  /**
+   * `normal` Change the row height of table
+   */
+  size: PropTypes.oneOf(['compact', 'short', 'normal', 'tall']),
```

Comment:
Should we include all of these options if they aren't currently being used? 👀

Reply:
Actually all are used. `<TableToolbar>` code translates `compact`/`short` table size to the small variant of tool bar, and `normal`/`tall` table size to the normal variant of tool bar.

Reply:
It seems like only `compact` and `short` are being used, correct? The other values would noop?

Reply:
In general, it seems like from the issue what we want to express is "display the small variant of toolbar if the table size is either compact or short"? Would it make sense for the size to then be either "small" or "normal" since that would refer to the height of the toolbar versus the size referring to the table? This logic could then come from the `DataTable` for determining the size in `getToolbarProps` if we want to use that

Reply:
Moved the size translation logic to `getToolbarProps()` as it seems to have been suggested: https://github.com/carbon-design-system/carbon/pull/5479/commits/fe23222e933410ab7c8e2543e27ad9f924c1d3a4